packages/README.md

```diff
@@ -10,3 +10,11 @@ in compliance with [Flutter repo style guide]( https://github.com/flutter/flutte
 1. Public getter
 2. Private field
 3. Public setter (when needed)
+
+## Naming for methods and functions
```

Comment:
in general, I don't think we need to rewrite a style guide for our own repo. We can link to Effective Dart and the Flutter style guide for references from this README.MD or from out CONTRIBUTING.MD file (this may be a better place), but it seems a bit overkill to try to pick and choose some of the style guidelines we follow and rewrite them here.

Reply:
We do not follow the entire Flutter style guide. Just some parts.

Reply:
Mentioned it.